test/integration/crcsuite/crcsuite.go

```diff
@@ -25,33 +25,149 @@ package crcsuite
 
 import (
 	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
 
 	"github.com/DATA-DOG/godog"
-	"github.com/code-ready/clicumber/testsuite"
+	"github.com/DATA-DOG/godog/gherkin"
+
+	clicumber "github.com/code-ready/clicumber/testsuite"
 )
 
-var sh testsuite.ShellInstance
+// FeatureContext defines godog.Suite steps for the test suite.
+func FeatureContext(s *godog.Suite) {
+
+	// CRC related steps
+	s.Step(`^removing CRC home directory succeeds$`,
+		CRCRemoveHome)
+
+	// CRC file operations
+	s.Step(`^CRC file "([^"]*)" exists$`,
```

Comment:
What is a 'CRC file' ? from the test step it is not clear what it is... but it sounds like a special type of file...

or is it just a file existence check fro the CRC home folder?

Reply:
> or is it just a file existence check fro the CRC home folder?
yes, the corresponding function is analogous to the FileExists from clicumber but prepends the location with `/path/to/.crc/`. Relative paths aren't processed well for some reason and I thought they look quite bad :) , e.g. `../../../../.crc` (in this case). If there's a better way to do it I'll change it right away.

Reply:
The naming is weird, as we generally call it the `CRC home folder`, etc. and therefore CRC file does not mean much, but rather 'File exists in the CRC home folder'

Reply:
@gbraad it's updated now. I also changed function names so they correspond to the steps.